src/processes/follow-topic-process.js

```diff
@@ -161,10 +206,12 @@ const followTopicProcess = async (job, done) => {
           //   });
         } else {
           // console.log(':::  Channel not eligible to receive auto message :::');
-          await chat.sendMessage({
-            text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id,
-            is_auto_message: true
+          await sendSignedMessage({
+            serverChannel: chat,
+            targetUserId: data?.user_id,
+            senderUsername: senderUser?.username,
+            communityMessageFormatUserId: communityMessageFormat.user_id,
+            text: communityMessageFormat?.message
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendSignedMessage` function is used correctly here to send a signed message if the channel is not eligible to receive an auto message.
```diff
+ await sendSignedMessage({
+   serverChannel: chat,
+   targetUserId: data?.user_id,
+   senderUsername: senderUser?.username,
+   communityMessageFormatUserId: communityMessageFormat.user_id,
+   text: communityMessageFormat?.message
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->